tests/README.md

```diff
@@ -27,14 +27,21 @@ compares it (string compare) with the expected result, a file named by the
 second argument. The `NAME` macro holds the test name,  which is `call_avg` in
 this case.
 
+These tests run as part of the normal suite of unit tests unless you are not
+running LLVM 12, in which case you need to install 'nix' and run these via this
+script `./tests/codegen-tests.sh`.
+
 #### Updating
 
-Run `./scripts/update_codegen_tests.sh` after making codegen changes up update
-the expected LLVM IR.
+**LLVM 12**
+
+If you are running LLVM 12 or if you need more control over which tests are updated,
```

Comment:
Unclear what "need more control" means here. I'm guessing you mean you can combine with `--gtest_filter` to only update a subset of codegen tests?

Reply:
Yup, updated the language to make it clearer.